wazo_setupd/plugins/setup/services.py

```diff
@@ -93,11 +93,13 @@ def get_engine_token(self, engine_password):
         )
         try:
             token_data = auth.token.new('wazo_user', expiration=60)
-        except HTTPError:
+        except HTTPError as e:
             raise SetupError(
                 message='Failed to create authorization token',
                 error_id='setup-token-failed',
-                details=self._auth_config,
+                details=dict(
+                    username='root', original_error=e, **no_service_key_auth_config
+                ),
```

Comment:
`Object of type HTTPError is not JSON serializable`
```suggestion
                details={
                    'username': 'root',
                    'original_error': e,
                    **no_service_key_auth_config,
                },
```

Reply:
Thanks, I did not know about the `{**mydict}` syntax!